app/main.py

```diff
@@ -2,0 +1,15 @@
+def calculate_team_total_rating(players: list) -> int:
+    player_reting = []
+    for player in players:
+        player_reting.append(player.get_rating())
+    return sum(player.get_rating() for player in players)
```

Comment:
The return statement already calculates the sum of player ratings directly. The list `player_reting` is not needed and can be removed.